ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -546,6 +546,8 @@ def _parse_data_model_columns(
         return columns
 
     def _get_database(self, database: str) -> Database:
+        if not database:
+            database = "default"
```

Comment:
Added this because
- database field is optional
- we don't handle multiple db support for the connector/dialect
- this will turn up as None - instead we can name it default

Reply:
cc - @harshach @pmbrull.
Merging this - if changes needed, will open up a new PR